doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -5412,13 +5412,35 @@ Example: ``curl -H "X-Dataverse-key: $API_TOKEN" -X POST  "https://demo.datavers
 
 This action changes the identifier of user johnsmith to jsmith.
 
-Make User a SuperUser
-~~~~~~~~~~~~~~~~~~~~~
+Toggle Superuser Status
+~~~~~~~~~~~~~~~~~~~~~~~
 
 Toggles superuser mode on the ``AuthenticatedUser`` whose ``identifier`` (without the ``@`` sign) is passed. ::
 
     POST http://$SERVER/api/admin/superuser/$identifier
 
+.. note:: This endpoint is deprecated. Use the set superuser status endpoint instead.
+
+.. _change-superuser-status:
+
+Set Superuser Status
+~~~~~~~~~~~~~~~~~~~~
+
+Specify the superuser status of a user with a boolean value.
+
+.. code-block:: bash
+
+  export SERVER_URL=http://localhost:8080
+  export USERNAME=jdoe
+  export IS_SUPERUSER=true
+  curl -X PUT "$SERVER/api/admin/superuser/$USERNAME" --data "$IS_SUPERUSER"
+
+The fully expanded example above (without environment variables) looks like this:
+
+.. code-block:: bash
+
+  curl -X PUT "http://localhost:8080/api/admin/superuser/jdoe" -d "true"
```

Comment:
Am I doing something wrong? When I do this...

    curl -X PUT "http://localhost:8080/api/admin/superuser/user4bad9d84" -d "true"

I get this error:

{"status":"ERROR","code":415,"message":"HTTP 415 Unsupported Media Type"}

If it matters, this is my version of curl:

```
$ curl --version
curl 8.1.2 (x86_64-apple-darwin22.0) libcurl/8.1.2 (SecureTransport) LibreSSL/3.3.6 zlib/1.2.11 nghttp2/1.51.0
Release-Date: 2023-05-30
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtsp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS GSS-API HSTS HTTP2 HTTPS-proxy IPv6 Kerberos Largefile libz MultiSSL NTLM NTLM_WB SPNEGO SSL threadsafe UnixSockets
```

That said, it works fine from AdminIT. 🤔 

But we need it to work from curl. That's how many people will use this API endpoint.

Reply:
It works if you add `-H "Content-Type: application/json"`:

```
pdurbin@air dataverse % curl -X PUT -d true -H "Content-Type: application/json" http://localhost:8080/api/admin/superuser/dataverseAdmin
{"status":"OK","data":{"message":"User @dataverseAdmin set as a superuser."}}
pdurbin@air dataverse % 
pdurbin@air dataverse % curl -X PUT -d false -H "Content-Type: application/json" http://localhost:8080/api/admin/superuser/dataverseAdmin
{"status":"OK","data":{"message":"User @dataverseAdmin removed as a superuser."}}
```

Reply:
`-H "Content-Type: text/plain"` works as well.

Reply:
I will add it to the doc!